packages/grafana-prometheus/src/querybuilder/components/PromQueryBuilder.test.tsx

```diff
@@ -98,7 +98,10 @@ describe('PromQueryBuilder', () => {
       labels: [{ label: 'label_name', op: '=', value: 'label_value' }],
     });
     await openMetricSelect(container);
-    await waitFor(() => expect(languageProvider.getSeries).toHaveBeenCalledWith('{label_name="label_value"}', true));
+    await waitFor(() =>
+      expect((languageProvider.getSeries as jest.Mock).mock.calls[0][1]).toEqual('{label_name="label_value"}')
```

Comment:
same suggestion as above, the same also applies to all other usages of `mock.calls[0][1]` in this file 😄

Reply:
https://github.com/grafana/grafana/pull/101889/commits/f16a50ef7463b94f5aa6f67e949b8fd0e45d9a2b